unity-client/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideoTexture.cs

```diff
@@ -214,10 +214,22 @@ private void CalculateVideoVolumeAndPlayStatus()
 
         private void UpdateVolume()
         {
-            if (texturePlayer != null)
-            {
-                texturePlayer.SetVolume(baseVolume * distanceVolumeModifier * Settings.i.generalSettings.sfxVolume);
-            }
+            if (texturePlayer == null) return;
+
+            float targetVolume = 0f;
+
+            if (CommonScriptableObjects.rendererState && IsPlayerInSameSceneAsComponent((CommonScriptableObjects.sceneID)))
```

Comment:
I'm not completely sure but `CommonScriptableObjects.rendererState` wont cast implicitly to `bool` in this case and it will just check if the property is null.

`CommonScriptableObjects.rendererState.Get()` should do the trick.

Reply:
Done